File: java-main/java-main/IO/Nestedfor5/p9.java
/*Write a program to take a number as input and print the Addition of Factorials of each
digit from that number.
*/

import java.util.Scanner;
class Demo{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = input.nextInt();
        int sum = 0;

        while(num > 0) {
            int digit = num % 10;
            sum += factorial(digit);
            num /= 10;
        }

        System.out.println("Addition of factorials of each digit = " + sum);
    }

    public static int factorial(int n) {
        if(n == 0 || n == 1) {
            return 1;
        }
        else {
            return n * factorial(n - 1);
        }
    }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session3/p9.c
/*  3
    6 9
    12 15 18

    4 
    8 12 
    16 20 24
    28 32 36 40*/

#include<stdio.h>
void main(){

	int rows;
	printf("Enter no of rows:\n");
	scanf("%d",&rows);


	for(int i=1;i<=rows;i++){
		int num=i;

		for(int j=1;j<=i;j++){

			printf("%d ",num);
			num++;
		}
		printf("\n");
	}
}

